_data/sites.json

```diff
@@ -4209,6 +4209,17 @@
         ]
     },
 
+    {
+        "name": "Mail.ru",
+        "url": "https://account.mail.ru/user/delete",
+        "difficulty": "impossible",
+        "notes": "You can deactivate your account and delete all data, but not entirely delete the account itself.",
+        "notes_ru": "Вы не можете удалить свою учётную запись, но можете удалить все её данные.",
+        "domains": [
+            "mail.ru"
+        ]
+    },
+
```

Comment:
`Mail.ru` should be before `mailbox.org` to keep the alphanumeric order and keep the Travis CI happy.

Reply:
> `Mail.ru` should be before `mailbox.org` to keep the alphanumeric order and keep the Travis CI happy.

Thanks